integration_tests/e2e/induction/createLongQuestionSetInduction.cy.ts

```diff
@@ -0,0 +1,43 @@
+import OverviewPage from '../../pages/overview/OverviewPage'
+import Page from '../../pages/page'
+import HopingToGetWorkValue from '../../../server/enums/hopingToGetWorkValue'
+import QualificationsListPage from '../../pages/induction/QualificationsListPage'
+
+context('Create a long question set Induction', () => {
+  beforeEach(() => {
+    cy.signInAsUserWithEditAuthorityToArriveOnPrisonerListPage()
+  })
+
+  it('should create a long question set Induction', () => {
```

Comment:
This test will be built upon with each PR. As we add each page we will update this test to form the complete journey